src/main/java/com/jaspersoft/jasperserver/jaxrs/client/core/JerseyRequest.java

```diff
@@ -34,9 +34,12 @@
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.client.Invocation;
 import javax.ws.rs.client.WebTarget;
+
+import javax.ws.rs.core.*;
```

Comment:
Wildcard import usage is prohibited by our internal code standards. Please setup your IDE to avoid it. Here is an example how to do so in Intellij IDEa:
http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports